test/symver/retX_new.c

```diff
@@ -12,9 +12,6 @@ for more details.  You should have received a copy of the GNU Lesser General
 Public License along with this program; if not, write to the Free Software
 Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 */
-
-int retX_new(int x) { return x; }
-__asm__(
-    "\t.globl  __retX_new\n"
-    "\t.equiv  __retX_new,retX_new\n"
-    "\t.symver __retX_new,retX@@GOTCHA_2");
+#include "sym_macro.h"
+SYMVER_ATTRIBUTE(retX_new, retX @ @GOTCHA_2)
```

Comment:
these spaces in `retX @ @GOTCHA_2` (instead of `retX@@GOTCHA_2`) are preserved by the preprocessor. I surprised, that the assembler removes them and produces the correct `retX@@GOTCHA_2`.

Reply:
The clang formatting was creating the issue. I fixed the formatting.